docs/content/specs-defs/specs/bicep/interfaces.md

```diff
@@ -219,7 +219,32 @@ An example of this is a Key Vault module that has a Private Endpoints enabled. I
 {{% /tab %}}
 {{< /tabs >}}
 
-## Secrets export
+## Secrets export (DEPRECATED)
```

Comment:
In case users wanted to backup secrets in a kv do we want to suggest them to do so at solution level instead and remove it from the module? Or do we want to keep the functionality inside the modules? Let's give a thought about pros and cons, also considering the compiled size for example.

Deprecated means we're not supporting the functionality anymore, so it would probably make more sense to slowly get rid of it before v.1.0

If instead we think it would be useful anyway to keep as an optional feature, maybe we could avoid the `deprecated` word and opt for something like `optional` for example?

I'd probably vouch for deprecating the feature, as long as we provide guidelines to users on how to transition